recipe/meta.yaml

```diff
@@ -10,17 +10,17 @@ source:
   sha256: bbbda47fd7eb04c0986df553f11aa0bdbc9846e75601b8ebd704fc8573bfb835
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.9
     - pip
   run:
-    - python >=3.8
-    - rasterio >=1.1.1
+    - python >=3.9
+    - rasterio >=1.2
     - scipy
```

Comment:
@snowman2, did you want to keep `scipy` as a dependency here, or remove it since it has been made optional since https://github.com/corteva/rioxarray/pull/416?

```suggestion
```

Reply:
We can leave it for now. Do you need it to be optional?

Reply:
Not really. Was just looking at the list and saw it was missing in the required dependencies.

Reply:
Makes sense. Thanks for bringing it up.